src/parts/Portal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { createPortal } from 'react-dom';
+
+interface PortalProps {
+  children: React.ReactNode;
+}
+
+const PORTAL_ROOT_ID = 'portal-root';
+
+const Portal = ({ children }: PortalProps): JSX.Element | null => {
+  const [portalElementNode, setPortalElementNode] = React.useState<HTMLElement>();
```

Comment:
Any thoughts on whether we should set `undefined` explicitly? I've always preferred it (i.e. `React.useState<HTMLElement | undefined>(undefined);` but I'm only about 2/10 on it, so open to not doing it as long as we're consistent.

Reply:
Yes, explicit is better in this case IMO. Again, I will set the undefined from now on in the new code :)